### PR TITLE
docs(i18n-id): Translate out.md until outFile.md into indonesian

### PR DESCRIPTION
**Pull request by 0length**
## Overview
This is part of #938 

## Changes
Translate for files:
- [x] out.md
- [x] outDir.md
- [x] outFile.md

### CODE REVIEW

**Review by mandaputtra — APPROVED** [review]
Sudah Bagus :+1: :100:

File: packages/tsconfig-reference/copy/id/options/outDir.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+```json tsconfig
+{
+  "compilerOptions": {
+    "outDir": "dist"
+  }
+}
+```
+
+Menjalankan `tsc` dengan pengaturan ini dapat memindahkan berkas-berkas ke folder `dist`:
+
```
Comment (mazipan) [inline packages/tsconfig-reference/copy/id/options/outDir.md:30]: folder: direktori

**Review by jefrydco — APPROVED** [review]

**Review by mazipan — CHANGES_REQUESTED** [review]

File: packages/tsconfig-reference/copy/id/options/outDir.md (hunk 1 above)
Comment (0length) [reply packages/tsconfig-reference/copy/id/options/outDir.md:30]: done

**Review by mazipan — APPROVED** [review]
LGTM 🎉

**Comment by mandaputtra** [comment]
Sepertinya botnya kalau di LGTM lewat komentar approve ga nyaut dia,

**Pull request merged**